scheduler/src/cook/queries.clj

```diff
@@ -28,3 +28,8 @@
             [?e :resource.type/mesos-name ?ident]]
           db)
        (map first)))
+
+(defn get-quota-resource-types
+  "Return a list of resource types used for quota, :cpus :mem :gpus ..."
+  [db]
+  (remove #{:disk} (get-all-resource-types db)))
```

Comment:
Instead of removing `:disk`, should we include the ones we do want to use for quota purposes?

Reply:
Yep, done